xcm/src/v0/junction.rs

```diff
@@ -52,6 +52,12 @@ pub enum BodyId {
 	/// The unambiguous judicial body (this doesn't exist on Polkadot, but if it were to get a "grand oracle", it
 	/// may be considered as that).
 	Judicial,
+	/// The unambiguous defence body (for Polkadot, an opinion on the topic given via a public referendum).
+	Defence,
+	/// The unambiguous administration body (for Polkadot, an opinion on the topic given via a public referendum).
+	Administration,
+	/// The unambiguous treasury body (for Polkadot, an opinion on the topic given via a public referendum).
```

Comment:
The comments in the parentheses are all wrong for all 3 of these variants.

Reply:
Can you advice?